config/app/manifest/en_US.json

```diff
@@ -1,12 +1,116 @@
 {
     "name": "Manifest",
     "provider": "Manifest",
-    "description": "",
-    "descriptionShort": "",
-    "configureUrl": "",
-    "categories": [],
-    "assets": [],
-    "resources": [],
-    "pages": [],
-    "label": []
-}
\ No newline at end of file
+    "description": "A short description to be shown below the app name with a promotional text",
+    "descriptionShort": "A long description explaining what the app does or talking about your business.",
+    "url": "https:\/\/url-not-visible-in-app-catalog-to-your-app-homepage.com",
+    "isAvailable": true,
+    "business_models": [
+        "** CHOOSE ONE OR MORE BELOW **",
+        "B2B",
+        "B2C",
+        "B2C_MARKETPLACE",
+        "B2B_MARKETPLACE"
+    ],
+    "categories": [
+        "** CHOOSE ONE OR MORE BELOW **",
+        "BI_ANALYTICS",
+        "CUSTOMER",
+        "LOYALTY",
+        "PAYMENT",
+        "PRODUCT_INFORMATION_SYSTEM",
+        "SEARCH",
+        "USER_GENERATED_CONTENT"
+    ],
+    "pages": {
+        "Overview": {
+            "MoreContent_ThisIsOptional": {
+                "title": "A text section with more content about the app",
+                "type": "text",
+                "data": "Free text with more content about the app. It will be shown below the long description section."
+            },
+            "Advantages_ThisIsOptional": {
+                "title": "You can also create a item list section",
+                "type": "list",
+                "data": [
+                    "Your app advantage #1",
+                    "Your app advantage #2",
+                    "Your app advantage #3"
+                ]
+            },
+            "AsMuchAsYouNeed_ThisIsOptional": {
+                "title": "The title of another text section",
+                "type": "text",
+                "data": "You can add as much content as you need. It can also be a list, just change the 'type' property and use an array here as shown above."
+            }
+        },
+        "Legal": {
+            "LegalText_ThisIsOptional": {
+                "title": "A text with legal content",
+                "type": "text",
+                "data": "Free text with legal content."
+            },
+            "AsMuchAsYouNeedToo_ThisIsOptional": {
+                "title": "The title of another section",
+                "type": "text",
+                "data": "You can add as much content as you need. It can also be a list, just change the 'type' property and use an array here as shown above."
+            }
+        }
+    },
+    "assets": [
+        {
+            "type": "icon",
+            "url": "\/assets\/images\/app_name\/logo.svg"
+        },
+        {
+            "type": "image",
```

Comment:
One image is enough. You also need to consider that people have to remove the things they don't need so we should need to keep it as lean as possible.

Reply:
the idea was to let people understand which file extensions are possible, but I'll figure out another way